NTUEELightDance/LightDance-Editor
# Editor #519 Clipboard

### What is this PR for?


### What type of PR is it?

[Feature]

### Todos

* [ ] Clipboard

### What is the Github issue?
"resolve #519 "

### How should this be tested?
Install blender add on and edit some colors & alpha,  then check copy & paste.
Please Press `ctrl + c`, `ctrl + v` in `View3D`.

### Screenshots (if appropriate)
Before:
![image](https://github.com/NTUEELightDance/LightDance-Editor/assets/111693712/9a612479-b092-4cbe-a086-e6477149c7c5)
After:
![image](https://github.com/NTUEELightDance/LightDance-Editor/assets/111693712/e74360a1-c076-49dd-b347-70a91c12de37)
